### PR TITLE
Fixed css and unavailable states

### PR DESCRIPTION
**Pull request by afreen23**
- Fixed extra space in data consumption header
 - Fixed chart label spacing
*Before*
data-cons-1

*After*
data-cons-2

 - Fixed unavailable state message
unavailable-fix-1
 - Fixed units format in breakdown card

### CODE REVIEW

File: frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card.scss [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
   display: flex;
-  margin: 0.6em 0em;
 }
 
 .nb-data-consumption-card__dropdown-item {
   margin-right: 0.4em;
 }
+
+.nb-data-consumption-card__chart-label {
+  margin-top: 1em;
```
Comment (rawagner) [inline frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card.scss]: please use PF variable
```suggestion
  margin-top: var(--pf-global--spacer--md);
```

Comment (afreen23) [reply frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card.scss]: Gotcha! Thanks!

**Pull request merged**